hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/missingContainers/missingContainers.tsx

```diff
@@ -192,16 +301,36 @@ export class MissingContainers extends React.Component<Record<string, object>, I
     this.setState({
       loading: true
     });
-    axios.get('/api/v1/containers/missing').then(response => {
-      const missingContainersResponse: IMissingContainersResponse = response.data;
-      const totalCount = missingContainersResponse.totalCount;
-      const missingContainers: IMissingContainerResponse[] = missingContainersResponse.containers;
+
+    axios.all([
+      axios.get('/api/v1/containers/unhealthy/MISSING'),
+      axios.get('/api/v1/containers/unhealthy/UNDER_REPLICATED'),
+      axios.get('/api/v1/containers/unhealthy/OVER_REPLICATED'),
+      axios.get('/api/v1/containers/unhealthy/MIS_REPLICATED')
```

Comment:
Let's try to get all unhealthy container information at once with:
```suggestion
    axios.all([
      axios.get('/api/v1/containers/unhealthy')
```

then change the logic accordingly. Hopefully this will be a small change.

Reply:
Getting this Response.

{"missingCount":0,"underReplicatedCount":0,"overReplicatedCount":0,"misReplicatedCount":0,"containers":[]} but not sure How will be Containers data and states so  Can we change this things in next JIRA I am  not sure about changes and logic unless I get proper data from cluster.

Reply:
In what kind of cluster did you get this response? Did you change everything accordingly to this change? If you don't pass any state as a parameter to the endpoint it will return all the containers (javadoc on the `@Path("/unhealthy/{state}")`: `Passing null returns all containers.`) . Based on the javadoc on the `@Path("/unhealthy")`:
```
   * Return
   * {@link org.apache.hadoop.ozone.recon.api.types.UnhealthyContainerMetadata}
   * for all unhealthy containers.
```
I am fine if we ask every states endpoint individually for now, but please change the `MISSING` state ask accordingly to the others (`/api/v1/containers/unhealthy/MISSING`).

Reply:
> Can we change this things in next JIRA I am not sure about changes and logic unless I get proper data from cluster.

Alright. Let's move the refactoring work to a follow-up JIRA.